index.html

```diff
@@ -0,0 +1,121 @@
+          <a href="#top-songs" class="close-menu">Top Songs</a>
+          <a href="#featured-performers" class="close-menu">Performers</a>
+          <a href="#partners" class="close-menu">Partners</a>
+        </div>
+      </div>
+      </div>
+    </header>
+
+    <section id="top-songs" class="top-songs">
+      <h2>Main program</h2>
+      <div class="musics-container">
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/speaker-icon.png" alt="">
+            <span class="music-author">Jazz</span>
+            <p class="music-description">Listen to the complex harmonies, rapid tempos, and intense disjunct melodic lines of the Afro-American folk music traditions, such as spirituals, work songs, and blues.</p>
+          </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/board-icon.png" alt="">
+            <span class="music-author">Opera </span>
+            <p class="music-description">Appreciate music in its entirety while the orchestra tells a story through  vocal pieces and instrumentals.  </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/card-icon.png" alt="">
+            <span class="music-author">Bands </span>
+            <p class="music-description">Never miss a beat with the well put together group of musicians playing popular music such as jazz, rock, or pop. </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/tickets-icon.png" alt="">
+            <span class="music-author">Tickets </span>
+            <p class="music-description">We deal with over a million tickets getting sold out each year around the world, so that fans never miss out on their favorite musicians that share their music.</p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/netwotk-icon.png" alt="">
+            <span class="music-author">Workshop</span>
+            <p class="music-description">We also have a workshop for aspiring and upcoming artists to showcase their work & express themselves</p>
+        </div>
+        </a>
+      </div>
+      <a class="events-btn" href="#">Checkout upcoming events </a>
+    </section>
+
+    <section id="featured-performers" class="featured-performers">
+      
+    </section>
+
+    <section id="partners" class="partners">
+      <h2>Partners</h2>
+      <div class="partners-container">
+        <img src="./images/pepsi-icon.svg" alt="pepsi-logo">
+        <img src="./images/espn-logo.svg" alt="espn-logo">
+        <img src="./images/puma-logo.svg" alt="puma-logo">
+        <img src="./images/mtv-live-seeklogo.com.svg" alt="mtv-logo">
+        <img src="./images/redbullenergydrink.svg" alt="red-bull">
+      </div>
+    </section>
```

Comment:
- [ ] As requested by the previous reviewer you should follow the template as much as possible. Please fix the second broken image in the speaker section and place the background image behind the speakers, Make sure your design follows the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/181620719-199bf397-6891-4c42-999e-18cdce45421d.png)

### Behance Template

![image](https://user-images.githubusercontent.com/83803414/181620784-8f2d05fc-ce63-411c-bf9b-74eda74eda41.png)